src/test/java/seedu/findvisor/logic/parser/ParserUtilTest.java

```diff
@@ -102,6 +103,13 @@ public void parsePhone_validValueWithWhitespace_returnsTrimmedPhone() throws Exc
         assertEquals(expectedPhone, ParserUtil.parsePhone(phoneWithWhitespace));
     }
 
+    @Test
+    public void parsePhone_validValueWithWhitespaceInBetween_returnsTrimmedPhone() throws Exception {
+        String phoneWithWhitespaceInBetween = VALID_PHONE_WITH_SPACE_BETWEEN;
+        Phone expectedPhone = new Phone(VALID_PHONE);
+        assertEquals(expectedPhone, ParserUtil.parsePhone(phoneWithWhitespaceInBetween));
+    }
+
```

Comment:
Perhaps this method can be removed as well if the `replaceAll()` method is removed

Reply:
I agree. I think the whitespace should not matter for phone numbers since we can stick to current constraints of only numeric characters?

Reply:
While the validation regex ensures the phone numbers can only be Singaporean numbers, keeping the `replaceAll()` method allows users to key in numbers in the form "9123 4567" which some people may prefer. It also removes multiple spaces to account for the case where users accidentally key in more than 1 space.